frontend/src/stores/redux/notification/notificationSlice.ts

```diff
@@ -0,0 +1,25 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit'
+import {RootState} from "../store";
+import {INotification} from "../../../types/notification";
+
+const initialState: INotification = {
+    show: false
+}
+
+export const notificationSlice = createSlice({
+    name: 'notification',
+    initialState,
+    reducers: {
+        showAlert: (state, action: PayloadAction<INotification>) => {
```

Comment:
Alert is about bad case. But your method is general for good and bad nofifications. 
```suggestion
        showNotification: (state, action: PayloadAction<INotification>) => {
```